src/main/java/org/apache/cassandra/sidecar/server/MainModule.java

```diff
@@ -383,19 +387,25 @@ public CassandraVersionProvider cassandraVersionProvider(DnsResolver dnsResolver
 
     @Provides
     @Singleton
+    @Named("StreamRequestRateLimiter")
     public SidecarRateLimiter streamRequestRateLimiter(ServiceConfiguration config)
     {
-        return SidecarRateLimiter.create(config.throttleConfiguration()
-                                               .rateLimitStreamRequestsPerSecond());
+        long permitsPerSecond = config.throttleConfiguration().rateLimitStreamRequestsPerSecond();
+        LOGGER.info("Configuring streamRequestRateLimiter with rateLimitStreamRequestsPerSecond={}",
+                    permitsPerSecond);
+        return SidecarRateLimiter.create(permitsPerSecond);
     }
 
     @Provides
     @Singleton
     @Named("IngressFileRateLimiter")
     public SidecarRateLimiter ingressFileRateLimiter(ServiceConfiguration config)
     {
-        return SidecarRateLimiter.create(config.trafficShapingConfiguration()
-                                               .inboundGlobalFileBandwidthBytesPerSecond());
+        long bytesPerSecond = config.trafficShapingConfiguration()
+                                    .inboundGlobalFileBandwidthBytesPerSecond();
+        LOGGER.info("Configuring ingressFileRateLimiter with InboundGlobalFileBandwidth={}/s",
```

Comment:
I think there is value to still log the raw value, i.e. byte/s. For instance, it is easier for analytics tool to extract the values with the same unit, instead of human readable formats. 

I would suggest this

```suggestion
LOGGER.info("Configuring ingressFileRateLimiter. inboundGlobalFileBandwidth={}/s rawInboundGlobalFileBandwidth={}/s",
```